src/test/acceptance/features/error-page.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Sever Error page
+  In order to facilitate the beta HTBHF programme
```

Comment:
Should we be using `beta`? I'm assuming we will still be using this code once the beta is complete?

Reply:
We have this in most of the feature files. I would leave it in for now as the program is beta.